hedera-node/src/main/java/com/hedera/services/state/expiry/renewal/RenewalRecordsHelper.java

```diff
@@ -63,11 +65,13 @@
 	public RenewalRecordsHelper(
 			final RecordStreamManager recordStreamManager,
 			final GlobalDynamicProperties dynamicProperties,
-			final Consumer<RunningHash> updateRunningHash
+			final Consumer<RunningHash> updateRunningHash,
+			final AliasManager aliasManager
 	) {
 		this.updateRunningHash = updateRunningHash;
 		this.recordStreamManager = recordStreamManager;
 		this.dynamicProperties = dynamicProperties;
+		this.aliasManager = aliasManager;
```

Comment:
We dont need aliasManager here I think, forCrypto is only called from streamCryptoRenewal/Removal methods where the graph ID is built form entity num. So it doesn't come from the txn directly.